cpp/cmake_modules/gRPCVariables.cmake

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Did you generate this file by `list-absl-libraries.sh`?
Could you add a warning comment to this file something like "DO NOT EDIT THIS FILE MANUALLY. YOU SHOULD UPDATE THIS FILE BY list-absl-libraries.sh ..."? (The warning comment should be generated by `list-absl-libraries.sh` automatically.)

Reply:
Will do!